src/ua/shalypenko/JDBC/JDBCMethods.java

```diff
@@ -0,0 +1,155 @@
+package ua.shalypenko.JDBC;
+
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class JDBCMethods {
+
+    public static String input(String message) {

```

Comment:
N: It's okay as one of many implementations, but in most cases, CRUD methods accept some data to work with using method parameters.
So, when you need to pass data, you just pass it to the method, not using some input() method.

Example: createPerson(Person person) <-
not: createPerson() {
... input()..
... input()..
}